dockers/Screenshotter/screenshotter.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This script does a one-shot creation of screenshots, creating needed
+# docker containers and removing them afterwards.  During development,
+# it might be desirable to avoid the overhead for starting and
+# stopping the containers.  Developers are encouraged to manage
+# suitable containers themselves, calling the screenshotter.js script
+# directly.
+
+for browserTag in firefox:2.46.0 chrome:2.46.0; do
+    browser=${browserTag%:*}
+    image=selenium/standalone-${browserTag}
+    echo "Starting container for ${image}"
+    container=$(docker run -d -P ${image})
+    [[ ${container} ]] || continue
+    echo "Container ${container:0:12} started, creating screenshots..."
+    res=Failed
```

Comment:
Because it's changed to `Done` one line down, *if* the run succeeds. If you consider this confusing, I can change it to be the other way round, but that probably wouldn't be any clearer either: the test isn't done or failed at that point.